### ISSUE TITLE
Problem with the zombie generator in Minecraft

### ISSUE THREAD

**Issue by alexangelus2024** [chocolate-edition/Public-Issue-Board#27]
I have encountered an issue with the zombie generator in the mod. Currently, the generator is not working as it should. Instead of spawning zombies regularly, every few seconds, red spores appear around the generator. I believe these spores are an indication that the generator is bugged or broken.

Even after waiting the usual time for zombie generation, no enemies appear, and the only thing visible are these red particles (spores), which suggest there is an error in the spawning process. 

I need help please!

If anyone else has experienced this problem or knows how to fix it, I would appreciate any suggestions or help.

**Comment by Crisptain** [comment]
Assuming you're referring to zombie *spawners* (spawner being the actual name of the block), they're explicitly designed to break after a couple of spawns in this pack, so that's probably what's going on. In other words, more than likely this is entirely intended and normal.

**Comment by alexangelus2024** [comment]
@Crisptain 
So I can't create experience farms with these spawns. Another problem I have is that RAIDS don't work either, the RAID signal appears but no one attacks.

**Comment by GildedToxin** [comment]
Correct, you wont be able to make farms with spawners. After 25ish spawns, they will stop, it is an intended feature

As for the raids, they are most likely spawning underground, check your surrounding caves!

**Comment by alexangelus2024** [comment]
@GildedToxin 
Thank you very much, I understand that there is no command or way to change this, right?

As for the raids, I will check it out, when I have the curse and the raid appears, I will go down to the caves, otherwise I will write again.

Thank you very much again for the help!

**Comment by GildedToxin** [comment]
@alexangelus2024 
Anytime! You can turn off the spawners functionality by removing the limited spawners mod

**Comment by alexangelus2024** [comment]
@GildedToxin 
If it's not too much trouble, could you tell me how to remove the mod from the server? Do I delete the mod folder?

**Comment by GildedToxin** [comment]
@alexangelus2024 If you are playing on a server youll have to remove it from both the server and your personal mod list

But yes, you go into the mods folder and search for the limited spawner mod, remove only that one from both server and personal/client

And you should be good!
